.github/workflows/test-python.yml

```diff
@@ -30,14 +30,14 @@ jobs:
           mongodb-replica-set: test-rs
```

Comment:
Don't we need to run the test suite on windows too?

Reply:
Updating

Reply:
That was easy 😅